src/js/components/sharedComponents/CheckboxTree.jsx

```diff
@@ -241,7 +243,7 @@ export default class CheckboxTree extends Component {
         }
         else if (!data.length) return null;
         return (
-            <div className="checkbox-tree">
+            <div className={`checkbox-tree ${className}`}>
```

Comment:
Wont this add an extra space if there is no className or even add `undefined` to the className? Can we change this to an if statement or method?

Reply:
What is the right way to do this? @ebdabbs I remember you did something here...?

Reply:
`const theClassName = className ? `checkbox-tree ${className}` :  'checkbox-tree';`

Reply:
or use CX